src/components/ContentHighlights/ContentHighlightsCardItemsContainer.jsx

```diff
@@ -28,6 +34,29 @@ const ContentHighlightsCardItemsContainer = ({
       </Alert>
     );
   }
+
+  const archivedContent = [];
+  const activeContent = [];
+  if (HIGHLIGHTS_ARCHIVE_MESSAGING) {
+    for (let i = 0; i < highlightedContent.length; i++) {
+      const {
+        courseRunStatuses,
+      } = highlightedContent[i];
+      if (courseRunStatuses) {
+        for (let j = 0; j < courseRunStatuses.length; j++) {
+        // a course is only archived if all of it's course runs are archived
+          if (courseRunStatuses[j] !== 'archived') {
+            activeContent.push(highlightedContent[i]);
+            break;
+          }
+          archivedContent.push(highlightedContent[i]);
+        }
```

Comment:
Is the goal here to only push the highlighted content into archivedContent once if all course runs in it are archived?  Right now, seems to be pushing the same highlighted content into archivedContent for each archived course run in it.